### PR TITLE
Create container configuration for Hugo

### PR DESCRIPTION
**Pull request by AarynSmith**
This creates a container configuration for Hugo static site generator

### CODE REVIEW

File: containers/hugo/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+## Summary
+
+*Develop static sites with Hugo, includes everything you need to get up and running.*
+
+|          Metadata           |                    Value                     |
+| --------------------------- | -------------------------------------------- |
+| *Contributors*              | [Aaryn Smith](https://gitlab.com/aarynsmith) |
+| *Definition type*           | Dockerfile                                   |
+| *Works in Codespaces*       | Yes                                          |
```
Comment (Chuxel) [inline containers/hugo/README.md:11]: I don't think VS Codespaces (or Codespaces in GitHub) works with Alpine based containers at the moment. Were you able to verify this works?

File: containers/hugo/.devcontainer/devcontainer.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+			// Rebuild the container if it already exists to update.
+			"VERSION": "latest",
+		}
+	},
+	// Set *default* container specific settings.json values on container create.
+	"settings": {
+		"terminal.integrated.shell.linux": "/bin/ash"
+	},
+	// Add the IDs of extensions you want installed when the container is created.
+	"extensions": [],
```
Comment (Chuxel) [inline containers/hugo/.devcontainer/devcontainer.json:21]: Would it make sense to install an extension for toml support for config.toml?

File: containers/hugo/.devcontainer/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    export VERSION=$(curl -s https://api.github.com/repos/gohugoio/hugo/releases/latest | grep "tag_name" | awk '{print substr($2, 3, length($2)-4)}') ;;\
+    esac && \
+    echo ${VERSION} && \
+    wget -O ${VERSION}.tar.gz https://github.com/gohugoio/hugo/releases/download/v${VERSION}/${VARIANT}_${VERSION}_Linux-64bit.tar.gz && \
+    tar xf ${VERSION}.tar.gz && \
+    mv hugo* /usr/bin/hugo && \
+    go get github.com/yaegashi/muslstack && \
+    muslstack -s 0x800000 /usr/bin/hugo
+
+FROM alpine:edge
```
Comment (Chuxel) [inline containers/hugo/.devcontainer/Dockerfile:20]: Just as a FYI, there's also a base image you could use here that has things like git, even zsh pre-installed.  Just reference `mcr.microsoft.com/vscode/devcontainers/base:alpine`. It's built out of the `alpine` container folder in the vscode-dev-containers repo and mainly runs [this script](https://github.com/microsoft/vscode-dev-containers/blob/master/script-library/common-alpine.sh). 

Not a requirement to use it, just a convenience if it applies.

File: containers/hugo/README.md (hunk 1 above)
Comment (AarynSmith) [reply containers/hugo/README.md:11]: You are correct, I thought I had worked when I tested it, but I guess it didn't. Updating now.

File: containers/hugo/.devcontainer/devcontainer.json (hunk 2 above)
Comment (AarynSmith) [reply containers/hugo/.devcontainer/devcontainer.json:21]: Makes sense. I've added Better-Toml. Updating Now.

File: containers/hugo/.devcontainer/Dockerfile (hunk 3 above)
Comment (AarynSmith) [reply containers/hugo/.devcontainer/Dockerfile]: My reference for the Hugo setup said that alpine:edge was required, but after testing the base image appears to work.

**Comment by Chuxel** [comment]
LGTM!

**Pull request merged**